omdata/orca/calc.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from shutil import which
+
+from ase.calculators.orca import ORCA, OrcaProfile
+from pydantic import Field
+from sella import Sella
+
+ORCA_FUNCTIONAL = "wB97M-V"
+ORCA_BASIS = "def2-TZVPD"
+ORCA_SIMPLE_INPUT = [
+    "EnGrad",
+    "RIJCOSX",
+    "def2/J",
+    "NoUseSym",
+    "DIIS",
+    "NOSOSCF",
+    "NormalConv",
+    "DEFGRID3",
+]
+ORCA_SIMPLE_INPUT_QUACC_IGNORE = [
+    "#slowconv",
+]
```

Comment:
Based on the discussion by @samblau, it seems like it may be worth I remove `slowconv` as a default anyway. We can keep this for now, but it's something on my mind.

Reply:
Definitely agree, no slowconv please